modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Panel/Panel.tsx

```diff
@@ -94,6 +95,17 @@ const PanelHeader: React.FC<IPanelHeaderProps> = ({
 					{title}
 				</h3>
 
+				{type !== 'regular' && (
```

Comment:
Because the tooltip text is specific to categorization, I think it is better to check if `type === 'categorization'`. This way, if new types other than `regular` are added in the future, the tooltip will not be displayed.